node/test/peer_strategies.js

```diff
@@ -146,12 +257,15 @@ allocCluster.test('prefer outgoing creates new conn even if incoming', {
     }
 });
 
-function setupEcho(channel, serviceName) {
+function setupEcho(channel, serviceName, cb) {
```

Comment:
Rename `cb` to `onEchoSent` to clarify that this is not a callback but handler that will be called after every response.